docs/synthesis.md

```diff
@@ -146,12 +146,35 @@ Also, if present in the telemetry, these attributes are also added to the entity
 
 [guid_spec]: guid_spec.md
 
+#### Prefixed tags
+
+This is a special section where specify prefixes.
+
+Every attribute name including any of the provided prefixes will get indexed as a tag, taking into account the following:
+* If there are multiple attributes matching against the same prefix, all of them will get indexed.
+* The prefix gets removed from the final tag name.
+* Prefixed attributes take the lower precedence, meaning that after removing the prefix, if there's already a tag with the same name coming from the same telemetry datapoint, it will be discarded.  
+
+> With a `label.` prefix as part of the `prefixedTags` list, and the telemetry containing any attribute prefixed with it,
+for instance, `label.name`, the final tag name will be `name`.
+
+Finally, notice the `prefixedTags` structure is not a nested child hanging from `tags`, but directly from `rules`.
+
+```yaml
+synthesis:
+  rules:
+    - identifier: entity.id
+      name: k8s.podName
+      prefixedTags:
+        example.:
+        example2_:
+```
```

Comment:
Is probably worth mentioning that this configuration doesn't override the `tags.` prefix